lib/endpoint/hostedSessions.ts

```diff
@@ -24,20 +25,55 @@ export type EnrichedHostedSession = HostedSession & {
   cancel: (auth?: string) => Promise<HostedSession>;
 };
 
-export interface HostedSessionCreateBody {
-  additional_services?: Nullable<AdditionalService[]>;
-  cancel_url?: NullableString;
-  destination?: Nullable<ArtaLocation>;
-  insurance?: Nullable<Insurance>;
-  internal_reference?: NullableString;
-  objects: ArtaObject[];
-  origin: ArtaLocation;
-  preferred_quote_types?: Nullable<QuoteType[]>;
-  preferred_parcel_transport_services?: Nullable<ParcelTransportServices[]>;
-  public_reference?: NullableString;
-  shipping_notes?: NullableString;
-  success_url?: NullableString;
-}
+export type HostedSessionCreateBody =
+  | {
+      additional_services?: Nullable<AdditionalService[]>;
+      cancel_url?: NullableString;
+      destination?: Nullable<ArtaLocation>;
+      insurance?: Nullable<Insurance>;
+      internal_reference?: NullableString;
+      objects: ArtaObject[];
+      origin: ArtaLocation;
+      preferred_quote_types?: Nullable<QuoteType[]>;
+      preferred_parcel_transport_services?: Nullable<ParcelTransportServices[]>;
+      public_reference?: NullableString;
+      shipping_notes?: NullableString;
+      success_url?: NullableString;
+      type?: 'booking';
```

Comment:
These four are optional for standard booking now also

```
public_instructions_location_quotes
public_instructions_booking_review
public_instructions_payment
public_instructions_confirmation
```

The only one that should be limited to `inbound_booking` is `public_instructions_object_details`